lib/tapioca/dsl/compilers/active_record_columns.rb

```diff
@@ -109,8 +110,7 @@ def decorate
 
           root.create_path(constant) do |model|
             model.create_module(AttributeMethodsModuleName) do |mod|
-              constant.columns_hash.each_key do |column_name|
-                column_name = column_name.to_s
+              constant._default_attributes.keys.each do |column_name|
```

Comment:
Can we use `attribute_names` instead, since it does not rely on the default values of attributes and is documented since Rails 6.1 (at least): http://api.rubyonrails.org/classes/ActiveRecord/AttributeMethods.html#method-i-attribute_names

```suggestion
              constant.attribute_names.each do |column_name|
```